### DISCUSSION TITLE
[Solved] Student Dev Pack's Free Copilot Not Showing Up on Subscriptions Page

### DISCUSSION THREAD

**Discussion by flatypus** [community/community#19063] [Copilot Conversations] [answered]
I joined Copilot’s beta half a year ago, but today I found out that it had been commercialized into a ‘free-trial’ type ordeal. However, it also says that students can have access to copilot for free, through the Student Development pack. I have also joined the Github Global Campus already and I can use the other features of the Student Development pack, but it doesn’t seem to let me use Copilot for free anywhere on the Subscriptions page. Is there any thing I’m missing, or is there anything I can do?

**Comment by petersteele111** [comment]
Same deal here. I have been on the student Github pack for a couple years now. Renewed it last march and they last 2 years at a time. I literally was just notified today that my copilot was not fully setup properly and that I needed to log in while in VS code. I have been using it for months without issue now. So I log in and it wants me to pay the 10 a month or 100 a year starting in August, despite still having access to the student pack for another year. I just went ahead and setup it up as the "free trial" until August, but a bit bummed since I was a beta tester and have been using it all this time without issue. Will it appear after august in our student packs or are we just screwed and have to shell out money for it?

**Comment by MetalTurtle18** [comment, marked as answer]
I was just in the same situation. Honestly have no idea what I did, but I went back to the sign up page and it told me I had free access because of my student pack. Specifically, it put me on [this page](https://github.com/github-copilot/free_signup), so that might work for you? Good luck!

**Reply by petersteele111** [reply]
Thanks! That worked for me as well!

**Reply by flatypus** [reply]
Thank you so much! That link worked for me!

**Reply by aqilc** [reply]
Thank you so much, this was the only thing that worked for me after a lot of searching!

**Reply by TakundaHwaire** [reply]
Just renewed my student pack and this worked. Thanks

**Reply by Noightmore** [reply]
Thank you, this does solve my issue!
